TrackerExtenstion.cs

```diff
@@ -38,14 +39,48 @@ public void Startup(StartupParams sp)
         /// </param>
         public void Ready(ReadyParams rp)
         {
+            // @todo Subscribe to an event after the Dynamo home screen is shown
+
+            // Display a MessageBox to the user
+            // @todo Use a view framework to improve the UI/UX
```

Comment:
or don't :) the current box does the trick nicely and has no overhead - unless the interaction needs to be more complex, with checkboxes, etc, i wouldn't add anything just for a message box.

Reply:
Discussion continues as issue https://github.com/teamtreedyn/Binoculars/issues/19 therefore marking this resolved.